### PR TITLE
New case study design DE

### PR DESCRIPTION
**Pull request by jutta-frieden**
This is to reflect the new case study design on our German splash pages. We are also adding a new case study with Gastrofix here.

### CODE REVIEW

**Comment by jutta-frieden** [comment]
@jamesshedden Could you have a look through the code for me please? Especially the new part on translations in story-page-new.js Thank you!

**Comment by jutta-frieden** [comment]
@georg911 @a-xin  Could one of you check the German for the new case study section (overview and case studies)?

File: app/pages/stories/stories/boost-capital.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import StoryPageNew from '../story-page-new';
+import StoryPageNew from '../story-page-new.en';
```
Comment (jamesshedden) [inline app/pages/stories/stories/boost-capital.js]: @jutta-frieden Could we rename these existing components to `StoryPageNewEn`, `StoryPageNewDe` etc? Will help make things clearer for people in future what part of the site they're working on — grab me if you need help with this!

File: app/pages/stories/story-page-new.js [hunk 2]
```diff
@@ -1,110 +1,21 @@
 import React from 'react';
-import Href from '../../components/href/href';
 import Page from '../../components/page/page';
-import Link from '../../components/link/link';
-import Message from '../../components/message/message';
-import IfLinkExists from '../../components/if-link-exists/if-link-exists';
+import StoryPageNewEn from './story-page-new.en';
+import StoryPageNewDe from './story-page-new.de';
+// import StoryPageNewFr from './story-page-new.fr';
+// import StoryPageNewEs from './story-page-new.es';
```
Comment (jamesshedden) [inline app/pages/stories/story-page-new.js]: @jutta-frieden Let's ✂️ these comments

File: app/pages/stories/story-page-new.js [hunk 3]
```diff
@@ -1,110 +1,21 @@
...
-            </p>
-            <Href to='signup.path' id='track-cta-sign-up' className='btn u-margin-Tm u-margin-Rm'>Sign up now</Href>
-            <Link to='contact_sales' className='btn btn--hollow u-margin-Tm'><Message pointer='cta.pro' /></Link>
-          </div>
-        </div>
+      <Page isInverted>
+        <StoryPageNewEn />
+        <StoryPageNewDe />
+        {/*<StoryPageNewFr />*/}
+        {/*<StoryPageNewEs />*/}
```
Comment (jamesshedden) [inline app/pages/stories/story-page-new.js]: @jutta-frieden Let's ✂️ these comments also 👍

File: app/pages/stories/stories.de.js [hunk 4]
```diff
@@ -1,87 +1,236 @@
...
+//             </StoryLink>
+//           </div>
+//         </div>*/}
+
+//         <hr className='u-margin-An' />
+//         <StartTakingPaymentsCTA />
+//       </Translation>
+//   );
+//   }
+// }
```
Comment (jamesshedden) [inline app/pages/stories/stories.de.js]: @jutta-frieden Let's ✂️ this commented-out block

**Comment by jamesshedden** [comment]
@jutta-frieden Noticed there's a slight layout issue now we only have 3 categories here, the tabs aren't quite centered:

image

Grab me tomorrow to figure this one out! 💭

File: app/pages/stories/stories.de.js [hunk 5]
```diff
@@ -1,87 +1,236 @@
...
-                      <div className='u-color-invert u-margin-Tm'>
-                        Tobias Friese, PicDrop
-                      </div>
-                    </div>
+        <div className="stories-industries">
+          <div className="site-container">
+            <div className="industry-filters grid">
+              { _.map(groupedStories, (stories, category) => {
+                return (
+                  <div className={tabClassesForCategory(category)} onClick={this.setActiveCategory.bind(this, category)} key={category}>
```
Comment (jamesshedden) [inline app/pages/stories/stories.de.js]: @jutta-frieden Noticed there's a slight layout issue now we only have 3 categories here, the tabs aren't quite centered:

image

Grab me tomorrow to figure this one out! 💭

**Comment by jamesshedden** [comment]
@jutta-frieden Given it a quick look and added a few comments, looking good! Let's go over tomorrow 👍

File: app/pages/stories/stories/boost-capital.js (hunk 1 above)
Comment (jutta-frieden) [inline app/pages/stories/stories/boost-capital.js]: Done 👍

**Comment by georg911** [comment]
@jutta-frieden looking good 🇩🇪

**Pull request merged**